internal/storage/gcs/bucket.go

```diff
@@ -20,6 +20,15 @@ import (
 	"golang.org/x/net/context"
 )
 
+// BucketType represent type of bucket like Hierarchical nonHierarchical as constants
```

Comment:
nit: -> "BucketType represents different types of buckets like Hierarchical or NonHierarchical as constants."